repository/t.json

```
@@ -3920,6 +3920,18 @@
 				}
 			]
 		},
+		{
+			"name": "Text-Mate Theme",
+  			"details": "https://github.com/vivekgohel2004/Text-Mate-Theme",
+  			"author": "vivekgohel2004",
+  			"labels": ["theme"],
```

```suggestion
			"name": "Text-Mate Color Scheme",
  			"details": "https://github.com/vivekgohel2004/Text-Mate-Theme",
  			"labels": ["color scheme"],
```

As the file extension indicates, in Sublime Text parlance your package is not a "theme" (which is for UI elements only) but a "color scheme" (syntax highlighting).

You don't need to specify the author if it's your GitHub username.

Finally, Text-Mate is a bit of an unfortunate name because [TextMate](https://macromates.com) is an editor that ST (and VS Code) borrows a lot of config file formats from. Like the `tmTheme` file format ("Text Mate Theme")... using the words "Text Mate" in your package name without it having a relation to the TextMate editor is going to be really confusing for a lot of people 😅